cmake/modules/FindGMIC.cmake

```diff
@@ -41,22 +42,43 @@ if(GMIC_LIBRARY AND GMIC_INCLUDE_DIR)
     message(STATUS "Found GMIC but version < ${GMIC_VERSION_NEEDED}. Compressed lut will not be available")
   else()
     if(WIN32)
-      if(GMIC_LIBRARY)
-        find_library(FFTW_LIBRARY NAMES fftw3)
+      find_library(FFTW_LIBRARY NAMES fftw3)
+      if(NOT FFTW_LIBRARY)
+        message(STATUS "missing GMIC dependency fftw3")
+      else()
         list(APPEND GMIC_LIBRARY ${FFTW_LIBRARY})
-      ENDIF(GMIC_LIBRARY)
+      endif(NOT FFTW_LIBRARY)
+      # workaround for msys2 gmic 9.0-3. Should be reviewed when gmic 9.3 is available
```

Comment:
Can we fix the version in the comment to be 2.9.0-3 to avoid confusion?
This is what shows up when I query the package manager for gmic.
$ pacman -Ss gmic
mingw32/mingw-w64-i686-gmic 2.9.0-3
    A Full-Featured Open-Source Framework for Image Processing (mingw-w64)
mingw64/mingw-w64-x86_64-gmic 2.9.0-3 [installed]
    A Full-Featured Open-Source Framework for Image Processing (mingw-w64)

Reply:
Good catch. I'll fix this.